libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -154,9 +156,16 @@ open class SalesforceSDKManager protected constructor(
     @JvmField
     protected val context: Context,
     mainActivity: Class<out Activity>,
-    loginActivity: Class<out Activity>?
+    private val loginActivity: Class<out Activity>? = null,
+    internal val nativeLoginActivity: Class<out Activity>? = null,
 ) : LifecycleObserver {
 
+    constructor(
+        context: Context,
+        mainActivity: Class<out Activity>,
+        loginActivity: Class<out Activity>? = null
+    ) : this(context, mainActivity, loginActivity, nativeLoginActivity = null)
+
```

Comment:
Adding the new param with a default value makes this a non-breaking change in Kotlin, but not Java unfortunately. The new constructor covers that case.